pyproject.toml

```diff
@@ -6,20 +6,24 @@ authors = ["Gamemode 4 Contributors"]
 
 [tool.poetry.dependencies]
 python = "^3.10"
-beet = "^0.103.1"
+beet = "0.104.1"
 mecha = "^0.87.0"
 bolt = "^0.45.0"
 PyYAML = "^6.0"
 pydantic = "^2.6.1"
 requests = "^2.28.1"
 pillow = "^10.0.0"
+repro-zipfile = "^0.1.0"
 
 [tool.poetry.dev-dependencies]
 black = "^22.6.0"
 isort = "^5.10.1"
 
 [tool.poetry.plugins.beet]
 commands = "gm4.commands"
+autoload = "gm4.plugins.autoload"
+
+[tool.beet]
```

Comment:
Is this correctly structured? Is `[tool.beet]` supposed to be empty?

Reply:
Yes this is required to be a stub for.. reasons I don't understand. See https://discord.com/channels/900530660677156924/900533796179419226/1009269844501352468